### PR TITLE
[FEATURE] Finish support reading of PRIDE SDRF 

### PR DESCRIPTION
**Pull request by jpfeuffer**
updated parse script. swith to 'path' for the first inputs to allow for URIs. add some more params

### CODE REVIEW

File: bin/parse_sdrf.py [hunk 1]
```diff
@@ -9,18 +9,18 @@
...
+  """Tool to convert sdrf files into OpenMS config files"""
 
 def openms_ify_mods(sdrf_mods):
   oms_mods = list()
 
   for m in sdrf_mods:
-    if "AC=UNIMOD" not in m:
-      raise Exception("only UNIMOD modifications supported.")
+    if "AC=UNIMOD" not in m and "AC=Unimod" not in m:
+      raise Exception("only UNIMOD modifications supported. " + m)
```
Comment (Zethson) [inline bin/parse_sdrf.py:23]: https://www.python.org/dev/peps/pep-0498/

Can highly recommend (even more useful with several interpolations and more complex scenarios).

Comment (jpfeuffer) [reply bin/parse_sdrf.py:23]: True. Actually, this script comes from upstream. I was thinking about how to best track its changes from @timosachsenberg. But I think it should be in a completely finished state soon, such that we barely would need to update.

Comment (jpfeuffer) [reply bin/parse_sdrf.py:23]: https://github.com/bigbio/sdrf-openms/blob/master/parse_sdrf.py

Comment (Zethson) [reply bin/parse_sdrf.py:23]: Well you can always just package it and put it on PyPi & Conda. Might be a little bit overkill for such a small script, but would help.

Comment (timosachsenberg) [reply bin/parse_sdrf.py:23]: Ok thanks for the Tipp! I will take a look at it.

**Pull request merged**